Classes/View/PreviewView.php

```diff
@@ -355,7 +355,8 @@ protected function drawGridColumn(array $row, Column $column)
                 $langPointer = $row['sys_language_uid'];
                 $localizeButton = $dblist->newLanguageButton(
                     $dblist->getNonTranslatedTTcontentUids($childrenUids, $dblist->id, $langPointer),
-                    $langPointer
+                    $langPointer,
+                    $colPosFluxContent
```

Comment:
I think this is where you want to use FluxService to read the virtual colPos value that's stored in the session. Otherwise, you'll be passing `18181` as colPos for every element on the page - which you can probably cause an error with by creating several containers with several columns and put children in each, then try to translate one of them. I expect this would affect all of them then.

Instead, if we can pass this virtual colPos and this then gets extracted in TceMain / ContentService when selecting the elements to translate, that should give the correct result.